resources/sync_gateway_configs/custom_sync/sync_gateway_log_rotation_cc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "logging": {
+    "default": {
+      "logFilePath": "/tmp/sg_logs/sg_log_rotation.log",
+      "logKeys": ["*"],
+      "logLevel": "debug",
+      "rotation": {
+        "maxsize": 1,
+        "maxage": 30,
+        "maxbackups": 2,
+        "localtime": true
+      }
+    }
+  },
+  "databases": {
+    "db": {
+      "server": "walrus:data",
```

Comment:
Please use the parametrized server url. We want all of our functional tests to run with a backing Couchbase Server for consistency.